### PR TITLE
782 shift index for all elements except web element

### PR DESCRIPTION
**Pull request by IgorGaidukEpam**

### CODE REVIEW

**Comment by IgorGaidukEpam** [comment]
> as I understand it's not ready to review, so I've just return it

Now it's ready already :D

File: jdi-light/src/test/java/io/github/epam/listtests/DataListTests.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        dataList.list().setWebElements(list);
+        assertEquals(dataList.get(1).getText(), element0Text);
+        assertEquals(dataList.get(2).getText(), element1Text);
+    }
+
+    @Test (expectedExceptions = RuntimeException.class)
+    public void jdiDataListNullTest() {
+        dataList = new DataList();
+        dataList.list().setWebElements(list);
+        dataList.get(0);
```
Comment (mf-code) [inline jdi-light/src/test/java/io/github/epam/listtests/DataListTests.java]: there is no any check here but it's a test

File: jdi-light/src/test/java/io/github/epam/listtests/JListTests.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        jList.list().setWebElements(list);
+        assertEquals(jList.get(1).getText(), element0Text);
+        assertEquals(jList.get(2).getText(), element1Text);
+    }
+
+    @Test (expectedExceptions = RuntimeException.class)
+    public void jdiJListNullTest() {
+        jList = new JList(locator);
+        jList.list().setWebElements(list);
+        jList.get(0);
```
Comment (mf-code) [inline jdi-light/src/test/java/io/github/epam/listtests/JListTests.java]: no checks here

File: jdi-light/src/test/java/io/github/epam/listtests/WebListTests.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        webList.setWebElements(list);
+        assertEquals(webList.get(1).getText(), element0Text);
+        assertEquals(webList.get(2).getText(), element1Text);
+    }
+
+    @Test (expectedExceptions = RuntimeException.class)
+    public void jdiWebListNullTest() {
+        webList = new WebList(locator);
+        webList.setWebElements(list);
+        webList.get(0);
```
Comment (mf-code) [inline jdi-light/src/test/java/io/github/epam/listtests/WebListTests.java]: no checks here

File: jdi-light/src/test/java/io/github/epam/listtests/DataListTests.java (hunk 1 above)
Comment (IgorGaidukEpam) [reply jdi-light/src/test/java/io/github/epam/listtests/DataListTests.java]: Here is @Test (expectedExceptions = RuntimeException.class) above this method;)

Comment (IgorGaidukEpam) [reply jdi-light/src/test/java/io/github/epam/listtests/DataListTests.java]: > there is no any check here but it's a test

Here is @test (expectedExceptions = RuntimeException.class) above this method;)

File: jdi-light/src/test/java/io/github/epam/listtests/JListTests.java (hunk 2 above)
Comment (IgorGaidukEpam) [reply jdi-light/src/test/java/io/github/epam/listtests/JListTests.java]: > no checks here

the same as above: expected exception

**Pull request merged**